src/PresentationalComponents/RulesTable/RulesTable.spec.ct.js

```diff
@@ -0,0 +1,642 @@
+    values: ['Reboot', 'Kernel'],
+    type: 'input',
+    filterFunc: (it, value) =>
+      it.description.toLowerCase().includes(value.toLowerCase()),
+    urlParam: 'text',
+    urlValue: (it) => it.replace(/ /g, '+'),
+  },
+  risk: {
+    selectorText: 'Total risk',
+    values: Array.from(cumulativeCombinations(Object.keys(TOTAL_RISK))),
+    type: 'checkbox',
+    filterFunc: (it, value) =>
+      _.map(value, (x) => TOTAL_RISK[x]).includes(it.total_risk),
+    urlParam: 'total_risk',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => TOTAL_RISK[x]).join(',')),
+  },
+  impact: {
+    selectorText: 'Impact',
+    values: Array.from(cumulativeCombinations(Object.keys(IMPACT))),
+    type: 'checkbox',
+    filterFunc: (it, value) =>
+      _.map(value, (x) => IMPACT[x]).includes(it.impact),
+    urlParam: 'impact',
+    urlValue: (it) => encodeURIComponent(_.map(it, (x) => IMPACT[x]).join(',')),
+  },
+  likelihood: {
+    selectorText: 'Likelihood',
+    values: Array.from(cumulativeCombinations(Object.keys(LIKELIHOOD))),
+    type: 'checkbox',
+    filterFunc: (it, value) =>
+      _.map(value, (x) => LIKELIHOOD[x]).includes(it.likelihood),
+    urlParam: 'likelihood',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => LIKELIHOOD[x]).join(',')),
+  },
+  category: {
+    selectorText: 'Category',
+    values: Array.from(cumulativeCombinations(Object.keys(CATEGORIES))),
+    type: 'checkbox',
+    filterFunc: (it, value) =>
+      _.intersection(
+        _.flatMap(value, (x) => CATEGORIES[x]),
+        it.tags
+      ).length > 0,
+    urlParam: 'category',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => CATEGORIES_MAP[x]).join(',')),
+  },
+  status: {
+    selectorText: 'Status',
+    values: STATUS,
+    type: 'radio',
+    filterFunc: (it, value) => {
+      if (value === 'All') return true;
+      else return it.disabled === (value === 'Disabled');
+    },
+    urlParam: 'rule_status',
+    urlValue: (it) => it.toLowerCase(),
+  },
+  impacting: {
+    selectorText: 'Systems impacted',
+    values: Array.from(cumulativeCombinations(Object.keys(IMPACTING))),
+    type: 'checkbox',
+    filterFunc: (it, value) => {
+      if (!value.includes('1 or more') && it.impacted_systems_count > 0)
+        return false;
+      if (!value.includes('None') && it.impacted_systems_count === 0)
+        return false;
+      return true;
+    },
+    urlParam: 'impacting',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => IMPACTING[x]).join(',')),
+  },
+};
+const filterApply = (filters) => applyFilters(filters, filtersConf);
+const filterCombos = [{ impacting: ['1 or more'] }];
+const filterData = (filters = DEFAULT_FILTERS, values = data) =>
+  filter(filtersConf, values, filters);
+
+//Columns should be in a separate file where we will store all the constants???
+export const columns = [
+  {
+    title: intl.formatMessage(messages.name),
+    transforms: [sortable, cellWidth(40)],
```

Comment:
`transforms` seems obsolete for the tests. why we have to use it?

Reply:
You are correct, removed it